### PR TITLE
Updated code to 0.3

### PR DESCRIPTION
**Pull request by MeenuKumari28**
Updated code to FTD-0.3

### CODE REVIEW

File: index.ftd [hunk 1]
```diff
@@ -1,112 +1,105 @@
...
 
 -- ftd.column:
-width: fill
-padding-horizontal: 50
-padding-horizontal if $is-mobile: 20
-padding-bottom: 200
-background-color: $fpm.color.main.background.base
+width: fill-container
+padding-horizontal.px: 50
+padding-horizontal.px if { ftd.device == "mobile" }: 20
```
Comment (amitu) [inline index.ftd]: @Arpita-Jaiswal Use `.responsive`?

File: index.ftd [hunk 2]
```diff
@@ -115,33 +147,41 @@ Body text here in h3
...
--- h3: Heading Small
+-- component h1:
+caption title:
+optional body body:
 
-Body text is here in h3
+-- ftd.column:
+width: fill-container
+region: h1
+;;append-at: inner -- TODO FTD 0.3
```
Comment (Arpita-Jaiswal) [inline index.ftd:164]: `append-at` is never going to come in `FTD 0.3`. Remove this line from everywhere

File: index.ftd [hunk 3]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
+padding-bottom.px: 16
+padding-top.px: 8
 
---- markdown:
-if: $body is not null
-body: $body
+-- end: ftd.row
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
```
Comment (Arpita-Jaiswal) [inline index.ftd:221]: Why `id` is needed here? The purpose it serves earlier is not there anymore.

File: index.ftd [hunk 4]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
+padding-bottom.px: 16
+padding-top.px: 8
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
+width: fill-container
+margin-bottom.px if {$h3.body != NULL}: -26
 
+-- markdown:
+if: {$h3.body != NULL}
```
Comment (Arpita-Jaiswal) [inline index.ftd:267]:
```suggestion
if: { h3.body != NULL }
```

File: index.ftd [hunk 5]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
+-- ftd.text: $h3.title
+role: $inherited.types.heading-medium
+color: $inherited.colors.text-strong
+padding-bottom.px: 16
+padding-top.px: 8
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
+width: fill-container
+margin-bottom.px if {$h3.body != NULL}: -26
```
Comment (Arpita-Jaiswal) [inline index.ftd:264]:
```suggestion
margin-bottom.px if { h3.body != NULL }: -26
```

File: index.ftd [hunk 6]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
 
---- markdown:
-if: $body is not null
-body: $body
+-- end: ftd.row
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
+width: fill-container
+margin-bottom.px if {$h2.body != NULL}: -26
```
Comment (Arpita-Jaiswal) [inline index.ftd:223]: Don't use `$` in FScript mode (i.e. inside `{}`). Also, use space after `{` and before `}`.
```suggestion
margin-bottom.px if { h2.body != NULL }: -26
```

File: index.ftd [hunk 7]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
 
+-- markdown:
+if: {$h2.body != NULL}
```
Comment (Arpita-Jaiswal) [inline index.ftd:226]:
```suggestion
if: { h2.body != NULL }
```

File: index.ftd [hunk 8]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
+-- ftd.text: $h4.title
+role: $inherited.types.heading-small
+color: $inherited.colors.text-strong
+padding-bottom.px: 16
+padding-top.px: 8
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
+width: fill-container
+margin-bottom.px if {$h4.body != NULL}: -26
```
Comment (Arpita-Jaiswal) [inline index.ftd:305]:
```suggestion
margin-bottom.px if { h4.body != NULL }: -26
```

File: index.ftd [hunk 9]
```diff
@@ -152,148 +192,125 @@ Body text is here in h3
...
--- ftd.column h3:
-caption title:
-optional body body:
-width: fill
-region: h3
-append-at: inner
-margin-top: 32
-margin-bottom: 16
+-- markdown:
+if: {$h4.body != NULL}
```
Comment (Arpita-Jaiswal) [inline index.ftd:308]:
```suggestion
if: { h4.body != NULL }
```

File: index.ftd [hunk 10]
```diff
@@ -304,39 +321,48 @@ body: $body
...
 
---- ftd.text: $key
+-- ftd.row:
+margin-bottom.px: 10
+spacing.px: 2
+
+-- ftd.text: $key-value.key
+
+-- ftd.text: $key-value.value
+if: {$key-value.value != NULL}
```
Comment (Arpita-Jaiswal) [inline index.ftd:360]:
```suggestion
if: { key-value.value != NULL }
```

File: index.ftd [hunk 11]
```diff
@@ -304,39 +321,48 @@ body: $body
...
 
---- ftd.text: $value
-if: $value is not null
+-- ftd.text: $key-value.default
+if: {$key-value.value = NULL}
```
Comment (Arpita-Jaiswal) [inline index.ftd:363]:
```suggestion
if: { key-value.value = NULL }
```

File: index.ftd [hunk 12]
```diff
@@ -346,22 +372,26 @@ if: $value is null
...
+margin-bottom.px: 16
+;;boolean open: true -- TODO FTD 0.3
+
+-- ftd.text:
+text: $h2-text.title
+role: $inherited.types.heading-small
+color: $inherited.colors.text
+padding-bottom.px: 8
+padding-top.px: 16
+;;$on-click$: toggle $open -- TODO FTD 0.3
```
Comment (Arpita-Jaiswal) [inline index.ftd:395]:
```suggestion
$on-click$: $ftd.toggle($a = $open)
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: index.ftd (hunk 1 above)
Comment (Arpita-Jaiswal) [reply index.ftd]: This is fixed @amitu.

**Pull request merged**